qemu/tests/qemu_guest_agent.py

```diff
@@ -492,6 +508,56 @@ def gagent_check_sync(self, test, params, env):
         error_context.context("Check guest agent command 'guest-sync'", logging.info)
         self.gagent.sync()
 
+    @error_context.context_aware
+    def gagent_check_container(self, test, params, env):
+        """
+        Start qga container behavior only for linux guest
+        and release version above rhel8.2.0 now:
+        steps:
```

Comment:
We should cancel this case when run with rhel8.0 and 8.1, or we can ignore this if maintainer thinks it is not necessary.

Reply:
For example, if it is  not support only for guest, you can write no RHEL.8.0 RHEL.8.1 in cfg.